strings/triforce.txt

```diff
@@ -102,3 +102,23 @@ You get to
 drink from
 the firehose
 -
+Do you prefer
+ bacon, pork,
+   or ham?
+-
+You get one
+wish.  Choose
+wisely, hero!
+-
+Now how fast
```

Comment:
not particular on this one

Reply:
Not sure which one you're pointing at; my best guess is the second of my suggestions.  Basically the idea behind that one was a sort of tl;dr of vanilla, since there are some similar strings around elsewhere.  But if you'd rather I cut that one, I'm fine with that.